playbooks/reboot_cluster.yml

```diff
@@ -46,3 +54,58 @@
     service:
       name: mysql
       state: started
+
+- hosts: compute:!controller
+  serial: 1
+  tasks:
+  - name: Reboot
+    command: shutdown -r now
+    async: 0
+    poll: 0
+    ignore_errors: true
+
+  - name: Wait for SSH to come back
+    wait_for: host={{ ansible_default_ipv4.address }}
+              port={{ ansible_ssh_port|default(22) }}
+              delay=15
```

Comment:
We should run a few of the checks that would exist on computes, such as nova-compute process, neutron linux bridge agent. We want to be sure the node is healthy before moving on.